mousestyles/path_diversity/path_features_advanced.py

```diff
@@ -0,0 +1,297 @@
+from __future__ import print_function, absolute_import, division
+import numpy as np
+import pandas as pd
+# from mousestyles.path_features import path_features
```

Comment:
I just saw what you did in path_features_advanced_proposed.py. That would work too!